ci/setup-build-test-gdk.ps1

```diff
@@ -34,61 +50,56 @@ class TestSuite {
-# There are basically two situations here: either we are trying to run tests, in which case we use EngineNetTest
-# Or, we try different build targets, in which case we use UnrealGDKTestGyms
-if (Test-Path env:BUILD_ALL_CONFIGURATIONS) {
-    $test_repo_url = "git@github.com:spatialos/UnrealGDKTestGyms.git"
-    $test_repo_relative_uproject_path = "Game\GDKTestGyms.uproject"
-    $test_project_name = "GDKTestGyms"
-
-    $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "EmptyGym", "$test_project_name", "TestResults", "SpatialGDK.", "$user_gdk_settings", $True, "$user_cmd_line_args")
+if ((Test-Path env:TEST_CONFIG) -And ($env:TEST_CONFIG -eq "Native")) {
+    # We run spatial tests against Vanilla UE4
+    $tests += [TestSuite]::new($gdk_test_project, "NetworkingMap", "VanillaTestResults", "/Game/Maps/FunctionalTests/SpatialNetworkingMap", "$user_gdk_settings", $False, "$user_cmd_line_args")
+    
+    if ($env:SLOW_NETWORKING_TESTS -like "true") {
+        $tests[0].test_results_dir = "Slow" + $tests[0].test_results_dir
+        
+        # And if slow, we run NetTest functional maps against Vanilla UE4 as well
+        $tests += [TestSuite]::new($native_test_project, "NetworkingMap", "NativeNetTestResults", "/Game/NetworkingMap", "$user_gdk_settings", $False, "$user_cmd_line_args")
+    }
 }
 else {
-    if ((Test-Path env:TEST_CONFIG) -And ($env:TEST_CONFIG -eq "Native")) {
-        $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "NetworkingMap", "$test_project_name", "VanillaTestResults", "/Game/SpatialNetworkingMap", "$user_gdk_settings", $False, "$user_cmd_line_args")
-    }
-    else {
-        $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "SpatialNetworkingMap", "$test_project_name", "TestResults", "SpatialGDK.+/Game/SpatialNetworkingMap", "$user_gdk_settings", $True, "$user_cmd_line_args")
-        $tests += [TestSuite]::new("$test_repo_url", "$test_repo_branch", "$test_repo_relative_uproject_path", "SpatialZoningMap", "$test_project_name", "LoadbalancerTestResults", "/Game/SpatialZoningMap",
-            "bEnableMultiWorker=True;$user_gdk_settings", $True, "$user_cmd_line_args")
-    }
-
+    # We run all tests and networked functional maps in single-worker mode
+    $tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "TestResults", "SpatialGDK.+/Game/Maps/FunctionalTests/SpatialNetworkingMap+/Game/Maps/FunctionalTests/SpatialZoningMap", "$user_gdk_settings", $True, "$user_cmd_line_args")
+    # And we run all Zoned tests in multi-worker mode
+    $tests += [TestSuite]::new($gdk_test_project, "SpatialZoningMap", "LoadbalancerTestResults", "/Game/Maps/FunctionalTests/SpatialZoningMap",
+        "bEnableMultiWorker=True;$user_gdk_settings", $True, "$user_cmd_line_args")
```

Comment:
```suggestion
        "$user_gdk_settings", $True, "$user_cmd_line_args")
```
I believe that with the new GDK changes, bEnableMultiWorker actually no longer exists as a GDK setting. Loadbalancing is default on, and is only controlled by whether the map's SpatialWorldSettings have Enable Multi Worker.

Reply:
Can be refactored even more to include the map in one run with the others (will make the tests a bit faster too).

Reply:
Tested and confirmed - GDK master does this; changed ;)

(have a failing test to re-enable in single-worker, will do Friday)